packages/data/tests/unit/adapters/facts/elide-test.ts

```diff
@@ -83,16 +70,6 @@ module('Unit | Adapter | facts/elide', function(hooks) {
   test('dataQueryFromRequest', function(assert) {
     const adapter = this.owner.lookup('adapter:facts/elide');
```

Comment:
Type this too

Reply:
Typescript complains when this is typed since we're accessing a private property to test it

Reply:
I think we've been getting around it in other contexts by using `thing['privateThing']`